egui-winit/Cargo.toml

```diff
@@ -49,6 +49,7 @@ tracing = "0.1"
 winit = "0.26.1"
 
 arboard = { version = "2.1", optional = true, default-features = false }
+smithay-clipboard = { version = "0.6.3", optional = true }
```

Comment:
We should put this under `[target.'cfg(target_os = "linux")'.dependencies]`

and then add a`#[cfg(target_os = "linux")]` to all uses of it, or
`#[cfg(all(target_os = "linux", feature = "smithay-clipboard"))]`

Reply:
I went for `cfg(unix)` everywhere instead as at least FreeBSD has [some kind of support](https://docs.freebsd.org/en/books/handbook/x11/#x-wayland) for Wayland.

Reply:
However it seems `winit` is using:
```
#![cfg(any(
    target_os = "linux",
    target_os = "dragonfly",
    target_os = "freebsd",
    target_os = "netbsd",
    target_os = "openbsd"
))]
```
instead for the unix platform module. I guess that means that compiling this as-is will fail on some esoteric operating system that is an unix but isn't any of those in the list.